daemons/gptp/common/common_port.cpp

```diff
@@ -811,6 +819,10 @@ void CommonPort::setAsCapable(bool ascap)
 		uint32_t local_clock, nominal_clock_rate;
 		FrequencyRatio local_system_freq_offset;
 		int64_t local_system_offset;
+		uint64_t mean_path_delay;
+
+      		getLinkDelay
```

Comment:
[Nitpick/formatting] 
Ahh! I found the misaligned line I was talking about y'day. I wish github showed me whether it was a tab/space.